tests/test_evaluate.py

```
@@ -1,9 +1,13 @@
+"""Tests for sleap_roots_training.evaluate module."""
+
 import pytest
 import tempfile
 import pandas as pd
 import numpy as np
 from pathlib import Path
-from unittest.mock import patch, MagicMock, Mock
+from unittest.mock import patch, MagicMock, Mock, mock_open
```

Ruff (0.12.2)

`unittest.mock.Mock` imported but unused

Remove unused import

(F401)